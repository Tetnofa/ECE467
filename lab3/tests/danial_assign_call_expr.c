int fcn2 () {
    
}
int g;
int main() {
    g = fcn2();
}