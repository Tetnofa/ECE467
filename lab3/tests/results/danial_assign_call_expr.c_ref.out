Program [useIO=0](1,0)
	Function[isProto=0]	Int(1,0)
		Identifier[name:fcn2](1,4)
		Scope(1,12)
	Scalar Declaration	Int(4,0)
		Identifier[name:g](4,4)
	Function[isProto=0]	Int(5,0)
		Identifier[name:main](5,4)
		Scope(5,11)
			Assignment(6,4)
				Reference(6,4)
					Identifier[name:g](6,4)
				CallExpr(6,8)
					Identifier[name:fcn2](6,8)
